pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "pacstall"
+version = "2.0"
```

Comment:
Should we put '2.0-dev" for now?

Reply:
Why?

Reply:
Because it’s a dev thingy I assume

Reply:
Resolved in https://github.com/pacstall/pacstall/discussions/379/#discussioncomment-1689282